pkg/agent/agent.go

```diff
@@ -122,14 +122,15 @@ func FlowsAgent(cfg *Config) (*Flows, error) {
 	}
 
 	ebpfConfig := &ebpf.FlowFetcherConfig{
-		EnableIngress: ingress,
-		EnableEgress:  egress,
-		Debug:         debug,
-		Sampling:      cfg.Sampling,
-		CacheMaxSize:  cfg.CacheMaxFlows,
-		TCPDrops:      cfg.EnableTCPDrops,
-		DNSTracker:    cfg.EnableDNSTracking,
-		EnableRTT:     cfg.EnableRTT,
+		EnableIngress:    ingress,
+		EnableEgress:     egress,
+		Debug:            debug,
+		Sampling:         cfg.Sampling,
+		CacheMaxSize:     cfg.CacheMaxFlows,
+		TCPDrops:         cfg.EnableTCPDrops,
+		DNSTracker:       cfg.EnableDNSTracking,
+		EnableRTT:        cfg.EnableRTT,
+		SupportOldKernel: cfg.SupportOldKernel,
```

Comment:
Actually, the agent could determine that on its own, without requiring a configuration?

Reply:
kernel versions upstream and downstream don't align that will make it hard for me to check on special kernel

Reply:
somewhere in comments you wrote: `For older kernel (< 5.17) kfree_sbk drop hook doesn't exists` so couldn't we just pick the kernel version and compare with 5.17 ?